src/config/localization/translations.json

```diff
@@ -6,7 +6,7 @@
   "Docs": "Docs",
   "Voting": "Voting",
   "Lottery": "Lottery",
-  "Connect Wallet": "Connect Wallet",
```

Comment:
Unless you're sure all `t("Connect Wallet")` are replaced, do not delete existing translation.  There are other places where `t("Connect Wallet")` is used.

Actually, let's do not change "Connect Wallet" to "Connect to Wallet", it seems to be a design error here.  Revert any changes related to "Connect to Wallet".